core/authentication_api.php

```diff
@@ -190,10 +190,11 @@ function auth_anonymous_account() {
 /**
  * Get the auth cookie expiry time.
  * @param boolean $p_perm_login Use permanent login.
+ * @param int $p_user_id The user id to get session expiry for.
```

Comment:
Concerning `@param` `int` vs. `integer`.
We don't have clear rules for that, but just one line below and most of the time`integer` is used.